lighthouse-core/audits/performance-budget.js

```diff
@@ -20,6 +20,8 @@ const UIStrings = {
     =1 {1 request}
     other {# requests}
    }`,
+  /** Table column header for how displaying how much larger the size of network requests were than a predetermined budget. */
```

Comment:
This desc is different than the ones in `i18n`.  Which format do you guys prefer?

Reply:
This seems a bit better to me to clarify the fact that it's a table column front and center 👍

Reply:
I personally find this format _less_ specific in general.  I also don't really see this as a huge problem because a lot fo files use both 🤷‍♂ 

Worth unifying?

Reply:
wait, @exterkamp, didn't you write this? Are you suggesting it and then shooting it down? :)

Reply:
I wrote it then had regrets, wondered which format was better #ragrets

Reply:
I like that we landed on the format in `18n.js`.  Resolving.